recipes/openvino/all/conanfile.py

```diff
@@ -301,14 +313,18 @@ def package(self):
         rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
 
     def package_info(self):
+        debug = "d" if self.settings.build_type == "Debug" and self.settings.os == "Windows" else ""
```

Comment:
the same for macOS actually
https://github.com/openvinotoolkit/openvino/blob/master/cmake/developer_package/OpenVINODeveloperScriptsConfig.cmake#L153